.travis.yml

```diff
@@ -12,78 +12,84 @@ os:
 env:
   global:
     - secure: "HIj3p+p2PV8DBVg/KGUx6n83KwB0ASE5FwOn0SMB9zxnzAqe8sapwdBQdMdq0sXB7xT1spJqRxuxOMVEVn35BNLu7bxMLfa4287C8YXcomnvmv9xruxAsjsIewnNQ80vtPVbQddBPxa4jKbqgPby5QhhAP8KANAqYe44pIV70fY="
-    - GH_DOC_BRANCH: develop
-    - GH_REPOSITORY: github.com/MDAnalysis/mdanalysis.git
-    - GIT_CI_USER: TravisCI
-    - GIT_CI_EMAIL: TravisCI@mdanalysis.org
-    - MDA_DOCDIR: package/doc/html
-    - MDA_OPTPACKAGES: opt/packages
+    - GH_DOC_BRANCH=develop
+    - GH_REPOSITORY=github.com/MDAnalysis/mdanalysis.git
+    - GIT_CI_USER=TravisCI
+    - GIT_CI_EMAIL=TravisCI@mdanalysis.org
+    - MDA_DOCDIR=package/doc/html
+    - MDA_OPTPACKAGES=opt/packages
+    - BUILD_DOCS=false
+    # Set default python version to avoid repetition later
+    - PYTHON_VERSION=2.7
+    - MAIN_CMD="./testsuite/MDAnalysisTests/mda_nosetests --with-coverage --cover-package MDAnalysis --processes=2 --process-timeout=400 --with-memleak --with-timer --timer-top-n 50 --no-open-files"
+    - SETUP=minimal
+    - COVERALLS=false
   matrix:
     - SETUP=minimal PYTHON_VERSION=2.7
-    - SETUP=full PYTHON_VERSION=2.7
+    - SETUP=full PYTHON_VERSION=2.7 COVERALLS=true
       #- SETUP=minimal PYTHON_VERSION=3.3
       #- SETUP=full PYTHON_VERSION=3.3
+
 matrix:
   fast_finish: true
   # allow_failures:
       #- env: SETUP=minimal PYTHON_VERSION=3.3
       #- env: SETUP=full PYTHON_VERSION=3.3
+  include:
```

Comment:
Yes. The include jobs are defined outside of the normal build matrix. For osx I would even suggest to just add a single minimal build here, since we want to test installation on OSX only.